test/api-test.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
```

Comment:
I realize you know likely know this already, but just a reminder that you need five additional tests for your routes per the Lab 8 instructions.